src/conf.py

```diff
@@ -99,7 +99,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.imgmath',
```

Comment:
@TotoGaz - in order for the light/dark theme to work properly with equations, I've turned off imgmath.  The built-in sphinx version seems to work just fine, tbh.